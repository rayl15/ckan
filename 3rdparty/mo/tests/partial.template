<h2>Names</h2>
{{#names}}
  {{> partial.partial}}
{{/names}}
