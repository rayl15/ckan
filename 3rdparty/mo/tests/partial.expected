<h2>Names</h2>
  <strong>Tyler</strong>

